src/components/CompareCard/CompareCard.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./CompareCard.scss";
+
+function CompareCard() {
+  return (
+    <>
+      <section>
+        <h3>Car Name here</h3>
+      </section>
+      <section>
+        <p>Fuel Type here</p>
+      </section>
+      <section>
+        <p>Engine</p>
+      </section>
+      <section>
+        <p>Torque</p>
+      </section>
+      <section>
+        <p>Acceleration</p>
+      </section>
+      <section>
+        <p>Top Speed</p>
+      </section>
+      <section>
+        <p>On Road Price</p>
+      </section>
+      <section>
+          <button>Remove</button>
+      </section>
```

Comment:
you can place them all inside one section tag. 
Section hasa special semantic meaning, if you are adding them just for some padding etc. I would suggest you to use div

Reply:
Okay @pashedin